### PR TITLE
06.Ansible.start Maxim Radaman

### PR DESCRIPTION
**Pull request by redman27**
06.Ansible.start Maxim Radaman

### CODE REVIEW

**Review by pluhin — CHANGES_REQUESTED** [review]
Your last home works copy of Ivan Ustinovich works

Please do your work more uniq

File: Maxim_Radaman/06.Ansible/index.html [hunk 1]
```diff
@@ -0,0 +1 @@
+Ivan_Ustinovich
```
Comment (pluhin) [inline Maxim_Radaman/06.Ansible/index.html:1]: :)

**Review by pluhin — CHANGES_REQUESTED** [review]

File: Maxim_Radaman/06.Ansible/index.html (hunk 1 above)
Comment (redman27) [reply Maxim_Radaman/06.Ansible/index.html]: oops

**Review by pluhin — APPROVED** [review]
Ok, accepted

**Pull request merged**